### PR TITLE
use config file

### PR DESCRIPTION
**Pull request by snowhork**
for #5

### CODE REVIEW

**Comment by snowhork** [comment]
@hirosassa Hi, please review.

File: README.md [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
+This is beta version.
 
-## install
+## Install
 ```
 go get -u github.com/snowhork/rdiam
 ```
 
-## usage
+## Required
```
Comment (hirosassa) [inline README.md]: `Requirement` is more common usage, IMO

File: cmd/root.go [hunk 2]
```diff
@@ -80,13 +86,80 @@ func initConfig() {
...
+
+	if conf.RedashEndPoint == "" {
+		return xerrors.New("RedashEndPoint is empty")
+	}
+	if conf.RedashUserAPIKey == "" {
+		return xerrors.New("RedashUserAPIKey is empty")
+	}
+	globalClient = redash.NewClient(conf.RedashEndPoint, conf.RedashUserAPIKey)
+	return nil
+
```
Comment (hirosassa) [inline cmd/root.go]: unnecessary newline?

**Review by hirosassa — APPROVED** [review]
Added a few minor comments, LGTM except that!

**Comment by snowhork** [comment]
Thanks!

**Pull request merged**